.vscode/settings.json

```diff
@@ -0,0 +1,11 @@
+{
+  "[javascript]": {
+    "editor.defaultFormatter": "rome.rome"
```

Comment:
Should be Biome now I guess right?

Incidentally I think the reason formatting wasn't working is because this is technically a separate package. Might be worth just creating a `biome.json` in `examples/microphone_nextjs` and setting up the formatting the same way as elsewhere.